cypher-shell/src/main/java/org/neo4j/shell/cli/CliArgHelper.java

```diff
@@ -167,6 +168,8 @@ private static ArgumentParser setupParser(ParameterMap parameterMap)
         connGroup.addArgument("-d", "--database")
                 .help("database to connect to. Can also be specified using environment variable " + ConnectionConfig.DATABASE_ENV_VAR)
                 .setDefault("");
+        connGroup.addArgument("-f", "--file")
+                .help("Pass a file with cypher statements to be executed.");
```

Comment:
Maybe add to the help that Cypher-shell will be shutdown after the file has been executed.